draft-reddy-cose-jose-pqc-hybrid-hpke.md

```diff
@@ -113,6 +113,10 @@ For readability the algorithm ciphersuites labels are built according to the fol
 
 The HPKE PQ/T hybrid ciphersuites for JOSE and COSE are defined in {{IANA}}. Note that the PQ/T Hybrid KEM in HPKE is not an authenticated KEM. The HPKE Base mode can only be supported with the PQ/T Hybrid KEM.
 
+# Key Management
+
+In HPKE JWE Key Encryption, when encrypting the Content Encryption Key (CEK) for multiple recipients, all recipients must use algorithms that support post-quantum cryptographic methods if any recipient requires post-quantum security. This applies to both PQC KEMs and Hybrid PQ/T KEMs. If one recipient uses a post-quantum algorithm, traditional algorithms (such as ECDH-ES) MUST NOT be used for any recipient. Allowing even one recipient to use a traditional algorithm while others use post-quantum algorithms would compromise the overall security of the system, as the traditional algorithm would remain vulnerable to quantum attacks. This requirement ensures that the encryption scheme is fully resistant to quantum threats across all recipients.
+
```

Comment:
I think requirement not to mix post-quantum and quantum-vulnerable algorithms is unnecessarily strict and would cause trouble in algorithm transition (this involves long-term keys). I think the proper requirement would be to not use quantum-vulnerable algorithms for messages that require post-quantum security.

Reply:
Updated text, please check.